src/stylesheets/component/_mainheader.scss

```diff
@@ -0,0 +1,85 @@
+
+.main-header{
+    width: 1043px;
+    height: 880px;
+    left: 0px;
+    top: 0px;
+    border-bottom-right-radius: 60%;
+/* Brand color/Secondary */
+
+    background: #1E3446;
+}
+.main-texth1{
+    position: relative;
+    top: 286px;
+    left: 104px;
+    width: 684px;
+    height: 216px;
+
+/* Heading 1 */
+
+    font-family: 'Quicksand';
```

Comment:
Can u move this to another place to reuse it? Something like `typography.scss` and put it into `base` folder